addons/mail/static/src/models/rtc/rtc.js

```diff
@@ -347,6 +353,7 @@ function factory(dependencies) {
             this._disconnectAudioMonitor && this._disconnectAudioMonitor();
             if (this.messaging.userSetting.usePushToTalk || !this.channel || !this.audioTrack) {
                 this.currentRtcSession.update({ isTalking: false });
+                await this._updateLocalAudioTrackEnabledState();
```

Comment:
Why this line now?

Reply:
I noticed an unwanted behaviour when testing, if you are speaking when changing from voice activation to push to talk, your microphone keeps broadcasting as if it was still voice activation until you cut it with the push to talk release.

It is because changing the push to talk option calls this function (to update how the isTalking is computed), the session is correctly set as not talking but we have to update the track enabled state accordingly.